src/main/kotlin/no/nav/omsorgspenger/aleneom/AleneOmOmsorgenService.kt

```diff
@@ -0,0 +1,31 @@
+package no.nav.omsorgspenger.aleneom
+
+import no.nav.omsorgspenger.CorrelationId
+import no.nav.omsorgspenger.Identitetsnummer
+import no.nav.omsorgspenger.Periode
+import no.nav.omsorgspenger.infotrygd.InfotrygdRammeService
+import no.nav.omsorgspenger.overføringer.AnnenPart
+
+internal class AleneOmOmsorgenService(
+        private val infotrygdRammeService: InfotrygdRammeService) {
+
+    internal fun hentAleneOmOmsorgen(
+            identitetsnummer: Identitetsnummer,
+            periode: Periode,
+            correlationId: CorrelationId): List<AleneOmOmsorgen> {
+         val rammemeldinger = infotrygdRammeService.hentAleneOmOmsorgen(
+                identitetsnummer = identitetsnummer,
+                periode = periode,
+                correlationId = correlationId
+        )
+
+        return rammemeldinger.map {
+            AleneOmOmsorgen(
+                    vedtatt = TODO(),
```

Comment:
Ja riktig. Vi mangler å parse `vedtatt` fra infotrygdtjenesten. Så må utvide interfacet `InfotrygdRamme` å legge til `vedtatt: LocalDate` - og er den vi her ønsker å mappe til `gjennomført = it.vedtatt()`